### DISCUSSION TITLE
Unknown target metric value for cpu utilization

### DISCUSSION THREAD

**Discussion by anmol-mnet** [kedacore/keda#4165] [Q&A / Need Help] [answered]
Version Details : 
gke : 1.24
keda. : 2.9

I am using rollout for one of my service and keda for autoscaling but when i am using cpu scaler with target metrics type : utilization, it is giving **unknown**  target value. 
* I have given resources : limits and requests values in all my containers
* I have checked my metric-adapter agent is giving cpu and memory metrics at node and pod level
* Scaledobject is showing no error, Ready and Active status is True.
* HPA is showing error : FailedGetResourceMetric  the HPA was unable to compute the replica count: no recommendation

But when i am using AverageValue metricType, hpa is able to calculate the target value. 

Can someone please suggest/help me with this.

**Comment by tomkerkhove** [comment, marked as answer]
Duplicate of #4164
